Update issue templates - Closes #1615 (#1616)
LiskHQ/lisk-docs: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Used, when solving of the issue requires the creation of new chapters and/or pages in the documentation, that did not exist before.
title: ''
labels: 'type: feature'
assignees: ''

---

### Motivation
Why is this new content needed?

> What is the intention/ motivation of creating it?

Reviewer: ```suggestion
What is the intention/motivation for creating it?
```

### Content structure
Describe the desired structure of the new content.

> Create a TOC as bullet list, and add more information about the desired content as possible:

Reviewer: ```suggestion
Create a TOC in the form of a bullet-point list, and add as much information as possible regarding the desired content:
```

- mention important information to include or emphasize on
- leave additional comments or suggestions, e.g. for creating diagrams

### Location

#### Affected branch / URL

> Choose the branch that should be used as base branch for this issue.

Reviewer: ```suggestion
Choose the applicable branch that should be used as the base branch for this issue.
```
